projects/libreoffice/fonts.conf

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<fontconfig>
+  <dir>/out/instdir/share/fonts/truetype</dir>
```

Comment:
This should hopefully work now. See https://github.com/google/oss-fuzz/blob/master/docs/fuzzer_environment.md